### ISSUE TITLE
Add a new organization to ROR: 
 
Pontifical Biblical Institute

### ISSUE THREAD

**Issue by mariagould** [ror-community/ror-updates#381]
Summary of request: Add a new organization to ROR

Name of organization: Pontifical Biblical Institute
Website: https://www.biblico.it/
Link to publications: https://punctumbooks.com/titles/the-old-nubian-texts-from-attiri/
Organization type: Education (A university or similar institution involved in providing education and educating/employing researchers)
Wikipedia page: https://en.wikipedia.org/wiki/Pontifical_Biblical_Institute
Wikidata ID: https://www.wikidata.org/wiki/Q951184
ISNI ID: https://isni.org/isni/000000012183685X
GRID ID: 
Crossref Funder ID: 
Other names for the organization:
Acronym/abbreviation:
Aliases: Biblicum
Labels:  Pontificium Institutum Biblicum*Latin ; Pontificio Istituto Biblico*Italian
Related organizations: 
City: Rome
Country: Italy
Geonames ID: 3169070
Year established: 1909
How will a ROR ID for this organization be used? To identify affiliations for research publications
Other information about this request:

**Comment by arthurpsmith** [comment]
I added Wikidata and ISNI id's.

Clearly a well-established and independent institute. I found a large number of references to it on Google scholar, including naming people who are or were professors there, but it seemed difficult to find use as an affiliation in scholarly literature. Here's one case though - a book review: https://www.jstor.org/stable/4489392
Anyway, I think it clearly qualifies for ROR, and should be added.
